backend/api/views/tests/test_resource.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from ...models.const import OhshownEventConst
```

Comment:
Not a problem but a note for our reference. The importing style differs in the test files under `api/views/tests/`, for example https://github.com/tai271828/disfactory-backend/blob/3ace2cf7333d0e5e04e59ffb345114cd005c9ef8/backend/api/views/tests/test_factory_report_record_r.py#L6 . We may want to unify them in the future.